src/lib/omerlo/reader/server/hooks.ts

```diff
@@ -9,25 +9,21 @@ import { refresh } from "./token";
 const handleApiProxy: Handle = async ({ event, ...tail }) => {
   event.url.host = env.PRIVATE_OMERLO_HOST;
   event.url.protocol = env.PRIVATE_OMERLO_PROTOCOL
-  event.url.port = ''; // This to prevent custom posts when working with localhost
```

Comment:
Please add an ENV. Otherwise, if the hub is localhost:5173, then port is set to 5173.